src/views/transactions/withdraw/withdraw.view.tsx

```diff
@@ -0,0 +1,289 @@
+import React from "react";
+import { connect } from "react-redux";
+import { useLocation } from "react-router-dom";
+import { push } from "connected-react-router";
+import { BigNumber } from "@ethersproject/bignumber";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+
+import { AppState, AppDispatch } from "src/store";
+import * as withdrawThunks from "src/store/transactions/withdraw/withdraw.thunks";
+import * as withdrawActions from "src/store/transactions/withdraw/withdraw.actions";
+import { changeHeader } from "src/store/global/global.actions";
+import useWithdrawStyles from "src/views/transactions/transfer/transfer.styles";
+import TransactionOverview from "src/views/transactions/components/transaction-overview/transaction-overview.view";
+import Spinner from "src/views/shared/spinner/spinner.view";
+import { AsyncTask } from "src/utils/types";
+import * as storage from "src/utils/storage";
+// domain
+import { EstimatedWithdrawFee } from "src/domain/";
+import {
+  Account,
+  HermezWallet,
+  FiatExchangeRates,
+  PoolTransaction,
+  Token,
+  Exit,
+  PendingDelayedWithdraw,
+} from "src/domain/hermez";
+import { PendingDelayedWithdraws } from "src/domain/local-storage";
+import { EthereumNetwork } from "src/domain/ethereum";
+import { account } from "src/persistence/parsers";
+
+interface WithdrawStateProps {
+  poolTransactionsTask: AsyncTask<PoolTransaction[], Error>;
+  step: withdrawActions.Step;
+  exitTask: AsyncTask<Exit | PendingDelayedWithdraw, Error>;
+  accountTask: AsyncTask<Account, string>;
+  estimatedWithdrawFeeTask: AsyncTask<EstimatedWithdrawFee, Error>;
+  isTransactionBeingApproved: boolean;
+  pendingDelayedWithdraws: PendingDelayedWithdraws;
+  ethereumNetworkTask: AsyncTask<EthereumNetwork, string>;
+  wallet: HermezWallet.HermezWallet | undefined;
+  preferredCurrency: string;
+  fiatExchangeRatesTask: AsyncTask<FiatExchangeRates, string>;
+  tokensPriceTask: AsyncTask<Token[], string>;
+}
+
+interface WithdrawHandlerProps {
+  onChangeHeader: (accountIndex: string) => void;
+  onGoToHome: () => void;
+  onLoadHermezAccount: (
+    accountIndex: string,
+    poolTransactions: PoolTransaction[],
+    fiatExchangeRates: FiatExchangeRates,
+    preferredCurrency: string
+  ) => void;
+  onLoadPoolTransactions: () => void;
+  onLoadExit: (
+    accountIndex: string,
+    batchNum: number,
+    completeDelayedWithdrawal: boolean,
+    pendingDelayedWithdraws: PendingDelayedWithdraw[]
+  ) => void;
+  onWithdraw: (
+    amount: BigNumber,
+    account: Account,
+    exit: Exit,
```

Comment:
Shouldn't  this also be `Exit | PendingDelayedWithdraw`?

Reply:
This has changed a little bit in the latest commits, but it has been solved.